### PR TITLE
#1101 Improving parameter reading from env.properties

### LINKED ISSUE #1101: Improving parameter reading from env.properties configuration
## Expected Behavior
When you read the configurations once from env.properties, you should not read the configuration from the file again but use the previously read configuration.

## Actual Behavior

At the moment, he reads the values every time from the file env.properties.

## Steps to Reproduce the Problem

1. Debugging class org.scada_lts.config.ScadaConfig.java

## Specifications

  - Version: v2.1.0
  - Platform: linux
  - Subsystem: open jdk 1.8, mysql 5.7, nodejs 8.12.0

## The problem reported by:
@mateuszhyski

### PR DESCRIPTION
**Pull request by grzesiekb**
I added private variables to the ScadaConfig class that store configuration values.
If they are read once using the methods:
1. public Boolean getBoolean (String propertyName, boolean defaultValue)
2. public Long getLong (String propertyName, int defaultValue)
3. public Integer getInt (String propertyName, int defaultValue)
4. public String getProperty (String propertyName)

The values of the corresponding properties will be saved in the variables that will then be used when reading these values for specific configuration properties.

And I used ScadaConfig to read the configuration for running scripts in the ScriptExecutor class

I assume that in the future we will create a collection in ScadaConfig that will store a value but for now because of the ease of implementation and time I decided to use a simpler implementation.

fix #1101

### CODE REVIEW

File: src/org/scada_lts/config/ScadaConfig.java [hunk 1]
```diff
@@ -137,41 +173,127 @@ public Properties getConf() {
...
 	 * Get property value of type long with default value
 	 * @param propertyName
-	 * @param defaultValues
+	 * @param defaultValue
 	 * @return
 	 */
-	public Long getLong(String propertyName, int defaultValues) {
-		Long result = new Long(defaultValues);
+	public Long getLong(String propertyName, int defaultValue) {
+		Long result = new Long(defaultValue);
```
Comment (Limraj) [inline src/org/scada_lts/config/ScadaConfig.java]: `Long result = Long.valueOf(defaultValue); `

Comment (grzesiekb) [reply src/org/scada_lts/config/ScadaConfig.java]: Resolved in 09ca4defa6a9faac2f993f0143191dc33680c677

File: src/org/scada_lts/config/ScadaConfig.java [hunk 2]
```diff
@@ -137,41 +173,127 @@ public Properties getConf() {
...
+		try {
+			if (MILLIS_SECONDS_PERIOD_UPDATE_UNSILENCED_ALARM_LEVEL.equals(propertyName) && millisSecondsPeriodUpdateUnsilencedAlarmLevel.isPresent()) {
```
[the remark this answers is not in the document]
Comment (grzesiekb) [reply src/org/scada_lts/config/ScadaConfig.java]: Non-optimal complexity for 5-10 elements in configuration.

[the remark this answers is not in the document]
Comment (Limraj) [reply src/org/scada_lts/config/ScadaConfig.java]: I'm sorry but can you be more specific? This would greatly simplify this code.

**Pull request merged**